index.html

```diff
@@ -67,6 +67,7 @@ <h2 class="subtitle">This site features full code examples using Apache Kafka®,
             <li><a href="error-handling/confluent.html">Uncaught Exceptions</a></li>
             <li><a href="optimize-producer-throughput/confluent.html">Optimize producer for throughput</a></li>
             <li><a href="produce-consume-lang/scala.html">Non-Java client applications</a></li>
+            <li><a href="multiple-event-type-topics/confluent.html">Multiple event types in a single topic</a></li>
```

Comment:
Just thinking outloud, should this live under `Build Applications` or `Kafka topics`?

Reply:
I think it should go in `Build Applications`, but I don't have a strong opinion.